### ISSUE TITLE
🔗 🚀 Feature Request: Post and Custom Post Carousel (Duplicate of #19405, Merged with #20201)

### ISSUE THREAD

**Issue by CordialGit** [elementor/elementor#20355]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

I created a **Recipe website** which was a **Custom Post Type**. Then I wanted a Template like this-- https://www.chelseasmessyapron.com/banoffee-pie-recipe/

Beautifully, I got it done with Elementor Pro, except that I couldn't get the Post Carousel at the Top of the Page with Elementor Pro.

The reason is because Elementor Pro doesn't yet have a **Post Carousel** that we can use to create both Posts and Custom Posts Carousel. 

So I had to use **e-Addons** for Post and Custom Post Carousel. But **e-Addons** is a heavy JavaScript Addon, and it slows down your website.

### Describe the solution you'd like

If Elementor can create a Post Carousel Widget-- which can be used to create Post and Custom Posts Carousel, then Elementor Pro would be my complete Suite, and I will kick e-Addons out of my website.

Waiting on Elementor to make the Post Carousel Widget available.

### Describe alternatives you've considered

I've considered **e-Addons** for Post and Custom Post Carousel. But **e-Addons** is a heavy JavaScript Addon, and it slows down your website.

I considered Unlimited Elements, Essential Addons and Premium Addons. But they are Pro solutions. Meaning that, to be able to use them to pull Data of Custom Post Type into a Post Carousel, I need to buy their Pro version.

You see, it’s difficult many times to buy Elementor Pro, then buy another Pro Addon again, for probably one small additional feature.

So I, as a Rational consumer, will be looking at a Win for myself, and so are others like me. We do not look for a Win for Elementor or an Elementor Addon maker. We look for a Win for ourselves.

In our own perspective of thinking, the winner must be us. That is, we the consumers. So we look for One(1) Pro solution that solves almost everything related to design that we need.

The Math is now this– Unlimited Elements, Essential Addons, Premium Addons and other Addons has Free Widgets. And Elementor has Free version as well. 

If I have to pay for Pro, or buy One extra Addon, then that Addon has to provide whatever else that I need-- which is missing from the Free version of Elementor.

So it’s a choice of choose One(1) from the Two(2)– where in our head, Elementor Pro is an Addon, and Other Addons is equally an Addon.

This is our Reference of reasoning. And I do hope you understand?

### Additional context

I created a Recipe website using Elementor Pro. 

The Elementor Posts widget cannot render Custom Templates. So I used **Elementor Custom Skin**(https://wordpress.org/plugins/ele-custom-skin) --- to render my Custom Template, as seen here-- https://prnt.sc/19lnm4p

But this Custom Template was rendered through Elementor Posts widget by extension capability of e-Addons.

Now, when I heard about Elementor's release of the **Loop Builder**, I knew that **Elementor Custom Skin** (https://wordpress.org/plugins/ele-custom-skin)--- will soon be kicked out of my website, and that will be a One plugin reduction.

And with the Loop Builder, I can create my own Custom Skin and Custom Templates. The thing is that I will still be stuck with e-Addons (that heavy JavaScript Elementor Addon that slows down your website).

The second thing is that a Carousel and Slider are alive. They scroll, they move, they depict life and activity. 

A template is just an inanimate Prototype that needs to receive life, and a Carousel is what gives it that life.

So Elementor Pro with Dynamic Tags, Loop Builder, and Post and Custom Posts Carousel, helps me to build websites that are alive and vibrant. Plus, the Posts Carousel will help me get rid of e-Addons out of my website, without needing to solve it with additional Addon cost.

That will be one more plugin reduction from my website, and Elementor Pro becomes my One Stop solution for all my website design needs.

**Comment by nicholaszein** [comment]
Hello @CordialGit!

Thank you for your submission! 🙏 

### 🔗 This issue is a Duplicate of #19405, and the requested feature will be solved by the Loop Builder.

Which are on the same subject. 

For this reason, I'm closing this issue and 🔃 **Merging** it with issue #20201, so we can have all the discussions in one location, and deal with it more efficiently.

Feel free to continue the conversation there. 😉 

### ✔️ Please, in the future, remember to check if there isn't an issue similar to yours already opened. 

Best regards,

Nick

**Comment by nicholaszein** [comment]
Hello there!

We have great news! 🙌 

### 📢 We're happy to announce that this feature was released with Elementor Core/Pro v3.11.0! 🥳 

You can now use the [Loop Carousel](https://elementor.com/help/loop-carousel/) widget.

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
